### DISCUSSION TITLE
Is it possible to set the background of a kitty window to an emoji?

### DISCUSSION THREAD

**Discussion by groves** [kovidgoyal/kitty#8042] [Q&A] [answered]
I'd like to dynamically set the background of inactive kitty windows to an emoji scaled to the size of the window based on the directory the active tab in the window is in. I see a window background image setting in kitty.conf. Is there a way to set the background to an emoji character so I don't need to create the images myself?

**Comment by kovidgoyal** [comment]
Why would you want to set the background to an emoji? use the window
logo feature instead. And these can be set via kitty's remote control
feature, as can background images.

**Reply by groves** [reply]
I think with window logo I need to have an existing image, no? With emoji, I don't need to generate images, I just need a unicode codepoint.

**Reply by groves** [reply]
(I'm trying to make inactive windows easily distinguishable so I know which one to click on without having to dig through the details in each one. I have a mapping from directories to an emoji to set a tab title already. I want to use that same emoji as the background when inactive to make them distinct. Really, being able to use kitty's text rendering to draw an arbitrary string at the full height of the window as a semi-transparent background is probably a better description of what I'm after)

**Comment by kovidgoyal** [comment, marked as answer]
On Tue, Nov 12, 2024 at 04:20:26AM -0800, Charlie Groves wrote:
> (I'm trying to make inactive windows easily distinguishable so I know which one to click on without having to dig through the details in each one. I have a mapping from directories to an emoji to set a tab title already. I want to use that same emoji as the background when inactive to make them distinct. Really, being able to use kitty's text rendering to draw an arbitrary string at the full height of the window as a semi-transparent background is probably a better description of what I'm after)

Ah, well there's no such facility. Though there are plenty of tools that
can render strings at arbitrary font sizes already, just use one of them
and the use the resulting imagea  window logo or background via remote
control.
